saga-core/src/test/java/io/servicecomb/saga/core/SagaStartTaskTest.java

```diff
@@ -0,0 +1,30 @@
+package io.servicecomb.saga.core;
+
+import static com.seanyinx.github.unit.scaffolding.AssertUtils.expectFailing;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+
+import org.junit.Test;
+
+public class SagaStartTaskTest {
+  private final SagaRequest request = mock(SagaRequest.class);
+  private final SagaLog sagaLog = mock(SagaLog.class);
+
+  private final SagaStartTask sagaStartTask = new SagaStartTask("0", null, sagaLog);
+
+  @Test
+  public void commit() {
```

Comment:
it's better to provide more meaningful name to reveal your intention of this test, eg. blowsUpWhenEventIsNotPersisted, so that it's easier for readers to understand the purpose of this test. you may refer to other tests.